tests/v2/validation/rbac/globalrolesv2/globalroles_v2.go

```diff
@@ -1,8 +1,11 @@
 package globalrolesv2
 
 import (
+	"context"
 	"fmt"
 	"strings"
+	"testing"
```

Comment:
why is testing added here, I don't see it being used in the new functions?

Reply:
Good catch! Testing was used here. but the helper should be updated to not use testing
(https://github.com/rancher/rancher/blob/7a51b55864feb93236f665c1a816879c55f565a2/tests/v2/validation/rbac/globalrolesv2/globalroles_v2.go#L230)